addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -165,25 +167,20 @@ export class LinkTools extends Link {
         super.focusUrl(...arguments);
     }
 
-    openDocumentDialog() {
-        this.props.wysiwyg.openMediaDialog({
-            resModel: "ir.ui.view",
-            useMediaLibrary: true,
-            noImages: true,
-            noIcons: true,
-            noVideos: true,
-            save: async (link) => {
-                this.initialNewWindow = this.initialIsNewWindowFromProps;
-                this._updateInitialNewWindowUI();
-                let relativeUrl = link.href.substr(window.location.origin.length);
-                await this._determineAttachmentType(relativeUrl.split("?")[0]);
-                if (this.isLastAttachmentUrl) {
-                    relativeUrl = relativeUrl.replace("&download=true", "");
-                }
-                this.$el[0].querySelector("#o_link_dialog_url_input").value = relativeUrl;
-                this.__onURLInput();
-            },
-        });
+    async uploadFile() {
```

Comment:
Stable retrocompatibility: Put back the original function name. Either keep the old code or redirect to this one, I'm not sure.

Reply:
Kept the old code (it would be weird to have method called `openDocumentDialog` that does not open a dialog :grimacing: )